src/app/controls/element-card/element-card.component.scss

```diff
@@ -0,0 +1,25 @@
+.card {
+  background-color: #f6f6f6;
+  width: 35rem;
+  border: #e0e0e0 1px solid;
+  border-radius: 8px;
+  .content{
+    height: 10rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
Can we use `justify-content: space-between` to remove the extra white space from bottom sides of the screen?
It is related to removing the` padding: 3rem` from the button component. Just to make the layout consistent and align vertically the whole content.

Reply:
fixed in different file